docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2950,28 +2953,40 @@ Create a volume
 
     {
       "Name": "tardis",
-      "Driver": "local",
+      "Driver": "custom",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
+      "Status": {
+        "hello": "world"
+      },
       "Labels": {
         "com.example.some-label": "some-value",
         "com.example.some-other-label": "some-other-value"
       },
+      "Scope": "local"
     }
 
 **Status codes**:
 
 - **201** - no error
 - **500**  - server error
 
-**JSON parameters**:
+**JSON parameters in request**:
```

Comment:
We can probably skip `in request` here, because we include "in response" below (and to keep this title consistent) wdyt @vdemeester?